policies/push-global.rego

```diff
@@ -1,75 +1,86 @@
-package spacelift
+# https://docs.spacelift.io/concepts/policy/git-push-policy
 
-# Update tracking for affected Terraform files
-track {
-    tf_affected
-    input.push.branch == input.stack.branch
-}
+package spacelift
 
-# Update tracking for affected stack config files
-track {
-    config_affected
-    input.push.branch == input.stack.branch
+# Ignore if any of the `ignore` rules evaluates to `true`
+ignore  {
+    not project_affected
+    not stack_config_affected
 }
 
-# Only trigger a stack run if the Terraform files were modified
-notrigger {
-  config_affected
-  not tf_affected
+ignore {
+    input.push.tag != ""
 }
 
-propose { tf_affected }
+# If pre-commit hooks make changes, they are not semantic changes and can and should be ignored
 ignore  {
-    not tf_affected
-    not config_affected
+    input.push.message == "pre-commit fixes"
 }
-ignore  { input.push.tag != "" }
 
-# If pre-commit hooks make changes, they are not semantic changes
-# and can and should be ignored.
-ignore  { input.push.message == "pre-commit fixes" }
-
-# Fetch all of our affected files
-filepath := input.push.affected_files
+# Only trigger a stack run if the project files were modified
+notrigger {
+    stack_config_affected
+    not project_affected
+}
```

Comment:
As I understand this, this rule will prevent the stack from being triggered of the only changes were to configuration files (stacks) that affect the stack. I thought the point of this PR was to _ensure_ that stacks were triggered when their configuration changes.

Reply:
this was in the module before (and it's not related to the changes in this PR).
I thought it's not correct, but I did not remove it because I might have missed some details which I don't know.

And yes, "the point of this PR was to ensure that stacks were triggered when their configuration changes.", but the `notrigger` was here before.

Reply:
The `notrigger` was here before because the update to the config via the mounted config file was going to do the trigger. We want to remove the `notrigger` here and probably remove the mounted config file altogether since we are now generating the config and runtime via `atmos`.